PR: Add FAQ questions on Spyder being open source and Anaconda situation (#205)
spyder-ide/spyder-docs: doc/faq.rst

      :alt: Navigator showing installing specific version of Spyder

.. dropdown:: Q: Is commercial use allowed? What's the situation with Anaconda?

   Spyder is 100% free and open source; there is no paid version or prohibition on commercial use.
   Spyder is developed by its international user community, and supported by its users through `OpenCollective`_ and by its generous sponsoring organizations, including `Quansight`_ and `NumFOCUS`_.

>    Our source code, standalone installers and most of our distribution methods (Pip/PyPI, Linux distros, MacPorts, WinPython, etc) are fully unencumbered and can be freely redistributed, used and modified by anyone.

Reviewer: I would suggest replacing the "unencumbered" word for something more familiar (specially for non native speakers).

>    Our source code, standalone installers and most of our distribution methods (Pip/PyPI, Linux distros, MacPorts, WinPython, etc) are fully unencumbered and can be freely redistributed, used and modified by anyone.

Reviewer: I would suggest replacing the "unencumbered" word for something more familiar (specially for non native speakers).

Author: ```suggestion
   Our source code, standalone installers and most of our distribution methods (Pip/PyPI, Linux distros, MacPorts, WinPython, etc) can be freely redistributed, used and modified by anyone, for any purpose, including commercial use.
```

Yeah, I don't know why I thought that was a good idea in a public FAQ for beginners from around the world. Ended up removing it completely and instead describing what I meant more clearly and explicitly.

   .. _OpenCollective: https://opencollective.com/spyder
   .. _Quansight: https://www.quansight.com/
   .. _NumFOCUS: https://numfocus.org/

>    The Anaconda distribution recently `recently changed`_ their EULA to add restrictions on large (> 200 employee) for-profit companies using it on a large scale, but that only applies to the package distribution infrastructure (the packaged Anaconda distribution and the ``defaults`` conda channel).

Reviewer: ```suggestion
   The Anaconda distribution recently `changed`_ their EULA to add restrictions on large (> 200 employee) for-profit companies using it on a large scale, but that only applies to the package distribution infrastructure (the packaged Anaconda distribution and the ``defaults`` conda channel).
```
Repeated word

Author: ```suggestion
   The Anaconda distribution `recently changed`_ their EULA to add restrictions on large (> 200 employee) for-profit companies using it on a large scale, but that only applies to the package distribution infrastructure (the packaged Anaconda distribution and the ``defaults`` conda channel).
```

Avoid breaking the link reference

>    Instead, you can simply download the similar `Miniconda distribution`_, which is 100% open source and identical to full Anaconda except for the gigabytes of Python packages installed by default in the Anaconda ``base`` environment (which we recommend you avoid using anyway).

Reviewer 2: ```suggestion
   Instead, you can simply download the similar `Miniconda distribution`_, which is 100% open source and identical to full Anaconda except for the many Python packages installed by default in the Anaconda ``base`` environment (which we recommend you avoid using anyway).
```

- The phrasing seems derogatory, which reads opinionated and not something I think we should include in our FAQ.
- Also, perhaps we should point directly to Miniforge instead of Miniconda?

Author: > The phrasing seems derogatory, which reads opinionated and not something I think we should include in our FAQ.

Agreed

> Also, perhaps we should point directly to Miniforge instead of Miniconda?

Absolutely, didn't even know that existed. I'll do it when I pull and rebase these in, since I'll need to change the link target as well

Author: Done; also modified the follow sentences to account for that change.

>    Then, simply install whatever packages you need (including Spyder, if you aren't using our recommended stand-alone installers) from the community-maintained ``conda-forge`` channel, which has a much wider variety of packages and is generally more up to date than the Anaconda ``defaults`` channel.

Reviewer 2: ```suggestion
   Then, simply install the packages you need (including Spyder, if you aren't using our recommended stand-alone installers) from the community-maintained ``conda-forge`` channel, which has a much wider variety of packages and is generally more up to date than the Anaconda ``defaults`` channel.
```

- I don't like `whatever` too much here.
- I think we should also add a link to the section in our Installation guide about our new installers.

Author: > I don't like whatever too much here.

Agreed

> I think we should also add a link to the section in our Installation guide about our new installers.

Good point, I'll do that in the rebase as well

Author: Added

   :container: + dropdown-id-commercial-use

   Spyder is 100% free and open source; there is no paid version or prohibition on commercial use.
   It is developed by its international user community, and supported by its users through `OpenCollective`_ and by its generous sponsoring organizations, including `Quansight`_ and `NumFOCUS`_.
   Our source code, standalone installers and most of our distribution methods (Pip/PyPI, Linux distros, MacPorts, WinPython, etc) can be freely redistributed, used and modified by anyone, for any purpose, including commercial use.
   For more details about the situation with Anaconda, see `that question`_.

   .. _OpenCollective: https://opencollective.com/spyder
   .. _Quansight: https://www.quansight.com/
   .. _NumFOCUS: https://numfocus.org/
   .. _that question: #anaconda-license

.. dropdown:: Q: What do the Anaconda licensing changes mean for Spyder?
   :container: + dropdown-id-anaconda-license

   If you use Spyder with the Anaconda distribution, they `recently changed`_ their `Terms of Service`_ to add restrictions on large (> 200 employee) for-profit companies using Anaconda on a large scale.

>    However, these terms only applies to the package infrastructure (the full Anaconda distribution and the ``defaults`` conda channel).

Reviewer: ```suggestion
   However, these terms only apply to the package infrastructure (the full Anaconda distribution and the ``defaults`` conda channel).
```
Isn't it like this?

Author: Yes indeed, good catch. I'll fixup it in.

Author: Done
